### PR TITLE
Document how to create custom routes for applications

### LINKED ISSUE #40: Document additional routes for applications
It was introduced here: https://github.com/epinio/epinio/issues/496

### PR DESCRIPTION
**Pull request by jimmykarily**
Fixes #40

### CODE REVIEW

File: src/howtos/custom-routes.md [hunk 1]
```diff
@@ -0,0 +1,42 @@
+# Custom Routes
+
+Epinio has the concept of a "system domain". This domain is set when installing
+Epinio (with the `domain` helm value). It must be a [wildcard domain](https://en.wikipedia.org/wiki/Wildcard_DNS_record) and is used in 3 different cases:
+
+- To create a URL for the Epinio API server
```
Comment (andreas-kupries) [inline src/howtos/custom-routes.md:6]:
```suggestion
- To create an URL for the Epinio API server
```

File: src/howtos/custom-routes.md [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+- To create a URL for the Epinio registry, when it's running inside the cluster (Optional)
```
Comment (andreas-kupries) [inline src/howtos/custom-routes.md:7]:
```suggestion
- To create an URL for the Epinio registry, when it's running inside the cluster (Optional)
```

File: src/howtos/custom-routes.md [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+- To create URLs for the applications deployed with Epinio (also known as "Routes")
+
+For example, if you deployed Epinio with `domain` set to `myawesomedomain.org`:
```
Comment (andreas-kupries) [inline src/howtos/custom-routes.md:10]:
```suggestion
For example, if Epinio was deployed with `domain` set to `myawesomedomain.org`:
```

File: src/howtos/custom-routes.md [hunk 4]
```diff
@@ -0,0 +1,42 @@
...
+
+- Epinio API would be accessed at `https://epinio.myawesomedomain.org`
```
Comment (andreas-kupries) [inline src/howtos/custom-routes.md:12]:
```suggestion
- The Epinio API would be accessible at `https://epinio.myawesomedomain.org`
```

File: src/howtos/custom-routes.md [hunk 5]
```diff
@@ -0,0 +1,42 @@
...
+- Epinio registry would be `https://epinio-registry.myawesomedomain.org`
```
Comment (andreas-kupries) [inline src/howtos/custom-routes.md:13]:
```suggestion
- The Epinio registry would be accessible at `https://epinio-registry.myawesomedomain.org`
```

File: src/howtos/custom-routes.md [hunk 6]
```diff
@@ -0,0 +1,42 @@
...
+
+and an application named "myapp" would be accessible at: `myapp.myawesomedomain.org`
+
+This requires zero setup for each application but it is sometimes needed to add
```
Comment (andreas-kupries) [inline src/howtos/custom-routes.md:17]:
```suggestion
This requires zero setup for each application. However, sometimes it is needed or desired to add
```

File: src/howtos/custom-routes.md [hunk 7]
```diff
@@ -0,0 +1,42 @@
...
+a custom domain, a dedicated one to your application. Let's assume the ap
```
Comment (andreas-kupries) [inline src/howtos/custom-routes.md:18]:
```suggestion
a custom domain, dedicated to the application. Let's assume that the application
```

File: src/howtos/custom-routes.md [hunk 8]
```diff
@@ -0,0 +1,42 @@
...
+above must also be accessible on `https://i-own-this-awesome-domain.org`. 
+
+You can add this domain to your application with the command below:
```
Comment (andreas-kupries) [inline src/howtos/custom-routes.md:21]:
```suggestion
Add this domain to your application with the command below:
```

File: src/howtos/custom-routes.md [hunk 9]
```diff
@@ -0,0 +1,42 @@
...
+
+```
+epinio app update sample --route i-own-this-awesome-domain.org
+```
+
+If you want to keep the original route too, you need to specify that explicitly:
```
Comment (andreas-kupries) [inline src/howtos/custom-routes.md:27]:
```suggestion
__Attention__, this removed the original route! To keep it, it has to be explicitly specified:
```

**Review by andreas-kupries — CHANGES_REQUESTED** [review]
Changed to more active voice.

File: src/howtos/custom-routes.md (hunk 1 above)
Comment (jimmykarily) [reply src/howtos/custom-routes.md:6]: I think a/an is chosen based on the "sound" of the next word not whether is a vowel or a consonant: https://whatis.techtarget.com/feature/Which-is-correct-a-URL-or-an-URL

Comment (andreas-kupries) [reply src/howtos/custom-routes.md:6]: Ok. I had it sounded out as `an url`, not as `a you-are-ell`, i.e. I did no speak spelling.

**Review by andreas-kupries — APPROVED** [review]

**Pull request merged**